common/api/geometry-core.api.md

```diff
@@ -2929,31 +2947,37 @@ export class InterpolationCurve3dOptions {
     clone(): InterpolationCurve3dOptions;
     cloneAsInterpolationCurve3dProps(): InterpolationCurve3dProps;
     get closed(): boolean;
+    set closed(val: boolean);
     static create(source: InterpolationCurve3dProps): InterpolationCurve3dOptions;
     get endTangent(): Vector3d | undefined;
+    set endTangent(val: Vector3d | undefined);
     get fitPoints(): Point3d[];
+    set fitPoints(val: Point3d[]);
     get isChordLenKnots(): number;
-    get isChordLenTangent(): number;

```

Comment:
Was renaming this `@public` property to `isChordLenTangents` intentional? Is it necessary? If so, please document the breaking change in NextVersion.md.